### DISCUSSION TITLE
Abrir una nueva tab haciendo click en un mensaje tipo text/html

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#719] [2. Questions and Answers]
Hola!
ande un mensaje tipo text/html en donde tengo un hipervínculo a una task, pero cuando la clickeo no se me abre una nueva tab con la task ):, sino que en el mismo mensaje se carga.

Alguien sabe cómo podría agregar el hipervínculo y que se abra en una nueva tab ? :o 

Así es cómo se ve:
![Screen Shot 2022-02-23 at 15 24 39](https://user-images.githubusercontent.com/42095617/155384043-125ba938-c649-4950-8cd3-b97492a50c1d.png)
![Screen Shot 2022-02-23 at 15 24 18](https://user-images.githubusercontent.com/42095617/155384065-d11b1880-e98f-46b4-8ec2-f9dd4b78b84d.png)

Y este es mi html que mando como mensaje:
```html
<table style="width:100%;font-family:sans-serif;">
<thead>
<tr><th style="text-align:left;">#</th><th style="text-align:left;">Órden de Trabajo</th><th style="text-align:left;">Costo Total</th></tr>
</thead>
<tbody> 
<tr> <td>17</td> <td> <a href="/tasks/61c3b3f45de9d200098a4615/list?groupId=61c3b3f45de9d200098a4615&task=62165868d703f4e582367916" target="_blank" >17 - [[EQUIPMENT-SECTOR-1]] - [[EQUIPMENT-1-A]]</a ></td> <td>$100,000.0</td> </tr>
 </tbody>
</table>"
```

**Comment by guillevalin** [comment]
La utilidad "DOM Purifier" elimina el tag "target" de los links.
Se puede usar esta página para probar los mensajes HTML: https://cure53.de/purify
